### ISSUE TITLE
Editor step input is confusing

### ISSUE THREAD

**Issue by lpolaright** [simeg/eureka#5]
So when you choose an editor it shows you this message:
```
What editor do you want to use for writing down your ideas?
1) vim (/usr/bin/vim)
2) nano (/usr/bin/nano)
3) Other (provide path to binary)
```

I wrote: `vim`, which produced the next message:
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ParseIntError { kind: InvalidDigit }', src/libcore/result.rs:906:4
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```

So, the issue for me was basically: 
I was thinking to just write vim, because with option #3 (providing the path) it looked like it's a string value.

Now that I faced that problem I know I need to put the integer.

**Comment by mchataigner** [comment]
why not using EDITOR env variable?
BTW, it's not possible to add some args to the editor

**Comment by simeg** [comment]
Fixed with PR #29.
